### DISCUSSION TITLE
Namibia Admin2 layer names not matching ground truth

### DISCUSSION THREAD

**Discussion by batinic-m** [wmgeolab/geoBoundaries#3481] [Q&A]
@DanRunfola it seems that the Namibia ADM2 dataset has admin labels mixed up. A couple of examples: 

Kamanjab Constituency:
![image](https://github.com/wmgeolab/geoBoundaries/assets/100201291/e8dc6982-b3cd-44ab-bc7d-65d95915faa9)
Labeled as Engela. Wiki: https://en.wikipedia.org/wiki/Kamanjab_Constituency

Engela Constituency:
<img width="278" alt="image" src="https://github.com/wmgeolab/geoBoundaries/assets/100201291/8aad849b-ebcd-4efa-8690-7084c9686512">
Labeled as Oshikango. Wiki: https://en.wikipedia.org/wiki/Engela_Constituency

Sesfontein Constituency:
<img width="256" alt="image" src="https://github.com/wmgeolab/geoBoundaries/assets/100201291/9fdbbf78-4646-4f66-9e4a-566ad9068607">
Labeled as Endola. Wiki: https://en.wikipedia.org/wiki/Sesfontein_Constituency

All of those screenshots are of geoboundaries Namibia AD2 layer geojson rendered in Qgis.

If you need additional information let me know. Do you maybe know when you could have a data drop with good naming?

Thanks

**Comment by DanRunfola** [comment]
@batinic-m Thanks for flagging this - we'll take a look and see how deep the problem goes (hopefully just a mis-joined attribute table, with some luck).

@jingjli1 can you put this high on our priority list for the Spring?

**Reply by batinic-m** [reply]
Thank you Dan! When you say Spring, what does that imply? Will the new drop be available March - April or May - June?

Thanks.
